src/libs/extra/source.liq

```diff
@@ -426,5 +430,6 @@ def source.cue(
     end
   end
 
-  source.on_track(s, handle_metadata)
```

Comment:
Not sure to understand here: why don't we use tracks and not metadata?

Reply:
Metadata can happen without a track mark. In most situations, e.g. when working with files, track marks and metadata happen at the same time but it's not guaranteed.

Reply:
I agree with that but I thought it would be more logical to generate cue entries for tracks rather than arbitrary metadata

Reply:
I'm trying to capture the most general use-case. I think metadata+deduplicate gives us the largest use-case cross-section..